tests/foreman/ui/test_oscapcontent.py

```diff
@@ -76,27 +73,6 @@ def test_negative_create_with_invalid_name(self):
                     self.assertIsNotNone(session.nav.wait_until_element(
                         common_locators['haserror']))
 
-    @tier1
-    def test_positive_default(self):
```

Comment:
That case seems irrelevant for a huge amount of time. Of course, we can see mentioned content even on latest 6.1.x, but not on any of 6.2. As oscap plug-in is not even default one for 6.2, I see no reason to raise that question further.